lint/file.go

```diff
@@ -202,24 +209,28 @@ func (f *File) disabledIntervals(rules []Rule) disabledIntervalsMap {
 			}
 
 			ruleNames := []string{}
-			if len(match) > 2 {
-				tempNames := strings.Split(match[3], ",")
-				for _, name := range tempNames {
-					name = strings.Trim(name, "\n")
-					if len(name) > 0 {
-						ruleNames = append(ruleNames, name)
-					}
+			tempNames := strings.Split(match[rulesPos], ",")
+			for _, name := range tempNames {
+				name = strings.Trim(name, "\n")
+				if len(name) > 0 {
+					ruleNames = append(ruleNames, name)
 				}
 			}
 
+			mustCheckDisablingReason := specifyDisableReason && match[directivePos] == "enable"
```

Comment:
Isn't this `disable`, or I'm missing something?

Reply:
ooops. Fixed